pubtools/_pulp/tasks/push/phase/push_post_actions.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import attrs
+
+from .base import Phase
+from . import constants
+
+from ....hooks import pm
+
+
+LOG = logging.getLogger("pubtools.pulp")
+
+
+class PostPushActions(Phase):
+    """Phase to do any neccesary PulpPushItem related operations
+
+    Currently it only triggers pulp_item_finished hook
+
+    Input queue:
+    - PulpPushItem items.
+
+    Output queue:
+    - none.
+    """
+
+    PROGRESS_TYPE = constants.PROGRESS_TYPE_NONE
+
+    def __init__(self, context, in_queue, **kwargs):
+        super(PostPushActions, self).__init__(
+            context, in_queue=in_queue, name="Post Push Actions", **kwargs
+        )
+
+    def run(self):
+        for item_batch in self.iter_input_batched():
+            for item in item_batch:
+                pm.hook.pulp_item_finished(
+                    item_metadata=attrs.asdict(item.pulp_unit)
```

Comment:
Could you please send the actual unit rather than converting to dict?  The unit is a properly documented class with stable API.  If you would prefer it as dict you can always convert it on the other side of the hook.

Reply:
Yeah, I can do that. I exported dicts as it's more memory efficient as push items can be freed after pulp push finishes.